source/binder.cpp

```diff
@@ -86,7 +99,9 @@ class ClassVisitor : public RecursiveASTVisitor<ClassVisitor>
 	virtual ~ClassVisitor() {}
 
 	virtual bool VisitEnumDecl(EnumDecl *record) {
+#if  (LLVM_VERSION_MAJOR >= 4)
```

Comment:
Why this ifdef needed?

Reply:
errs() is not present in old llvm, I think.